lib/msal-browser/src/app/PublicClientApplication.ts

```diff
@@ -73,8 +77,12 @@ export class PublicClientApplication extends ClientApplication implements IPubli
      * @returns A promise that is fulfilled when this function has completed, or rejected if an error was raised.
      */
     loginPopup(request?: PopupRequest): Promise<AuthenticationResult> {
-        this.logger.verbose("loginPopup called");
-        return this.acquireTokenPopup(request || DEFAULT_REQUEST);
+        const correlationId: string = this.getRequestCorrelationId(request);
```

Comment:
Can we make correlationId required in the InteractionClient constructors if we're changing things to ensure they'll be set before then?

Reply:
Yeah, I can look at that.

Reply:
The pairwise broker depends on `StandardInteractionClient` (well, things that subclass it), and changing correlation id to required would be technically breaking, so I'll leave it as is for now, and potentially do that incrementally.

Reply:
Sounds good, I'm fine with this being done incrementally - there shouldn't be anything we can change in the public repo that breaks the 1P library, however, since 1P should be pinning its dependency to a specific version of msal-browser